conf/keycloak/open-products-facts-realm.json

```diff
@@ -0,0 +1,2407 @@
+          "composite": true,
+          "composites": {
+            "client": {
+              "account": [
+                "view-consent"
+              ]
+            }
+          },
+          "clientRole": true,
+          "containerId": "5ca80ba2-8030-4f5f-909c-dab3221dc652",
+          "attributes": {}
+        },
+        {
+          "id": "71219f0a-0ba0-4521-a14e-a7d44a0b18ee",
+          "name": "manage-account",
+          "description": "${role_manage-account}",
+          "composite": true,
+          "composites": {
+            "client": {
+              "account": [
+                "manage-account-links"
+              ]
+            }
+          },
+          "clientRole": true,
+          "containerId": "5ca80ba2-8030-4f5f-909c-dab3221dc652",
+          "attributes": {}
+        },
+        {
+          "id": "af2495f7-4698-4e29-a3c8-d4a5b8ef0a99",
+          "name": "manage-account-links",
+          "description": "${role_manage-account-links}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "5ca80ba2-8030-4f5f-909c-dab3221dc652",
+          "attributes": {}
+        },
+        {
+          "id": "c441a16c-59ef-43bc-9526-4e81f09af158",
+          "name": "view-consent",
+          "description": "${role_view-consent}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "5ca80ba2-8030-4f5f-909c-dab3221dc652",
+          "attributes": {}
+        },
+        {
+          "id": "5a4b1f2e-3771-4ad5-8d5f-e79e19e7f2e6",
+          "name": "view-groups",
+          "description": "${role_view-groups}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "5ca80ba2-8030-4f5f-909c-dab3221dc652",
+          "attributes": {}
+        },
+        {
+          "id": "9271fdb4-6c46-43dd-b1bf-288211bf786f",
+          "name": "view-applications",
+          "description": "${role_view-applications}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "5ca80ba2-8030-4f5f-909c-dab3221dc652",
+          "attributes": {}
+        }
+      ]
+    }
+  },
+  "groups": [],
+  "defaultRole": {
+    "id": "6254728b-ae51-4873-bcb3-3d9854845b00",
+    "name": "default-roles-open-products-facts",
+    "description": "${role_default-roles}",
+    "composite": true,
+    "clientRole": false,
+    "containerId": "793a2761-1af2-44e1-a0b8-cc37a030a2af"
+  },
+  "requiredCredentials": [
+    "password"
+  ],
+  "passwordPolicy": "passwordHistory(2) and notUsername(undefined) and notEmail(undefined) and length(12)",
+  "otpPolicyType": "totp",
+  "otpPolicyAlgorithm": "HmacSHA512",
+  "otpPolicyInitialCounter": 0,
+  "otpPolicyDigits": 8,
+  "otpPolicyLookAheadWindow": 1,
+  "otpPolicyPeriod": 30,
+  "otpPolicyCodeReusable": false,
+  "otpSupportedApplications": [
+    "totpAppFreeOTPName"
+  ],
+  "localizationTexts": {},
+  "webAuthnPolicyRpEntityName": "Open Products Facts",
```

Comment:
To be honest, I'm not sure if we need to use this exact config to set Keycloak in prod. Might have to change some parameters, anyways, because of differences when running in developer mode and integrated H2 database, and probably PostgreSQL in production.